src/content/upgrades/merge/index.md

```diff
@@ -19,6 +19,8 @@ This process is in its final stages, with plans to undergo The Merge on a few pu
 
 ## What is The Merge? {#what-is-the-merge}
 
+<iframe width="1905" height="315" src="https://www.youtube.com/embed/EEuPmA8w0Kc" title="ETHEREUM MERGE - The Most Anticipated Event In Crypto Explained" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
```suggestion
<YouTube id="EEuPmA8w0Kc" />
```
Nice addition. We can just use the YouTube component that @wackerow made here instead of the iframe.